src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The Storage class saves and loads tasks from the save file.
+ */
+public class Storage {
+    /** The path of the file to read/write */
+    private Path filepath;
+
+    /**
+     * Constructor for Storage.
+     *
+     * @param filepath The filepath of the storage file.
+     */
+    public Storage(Path filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Loads from the save file if available.
+     * Returns an array of strings containing the task,
+     * throw DukeException if file is not found.
+     *
+     * @return ArrayList containing string representation of the tasks.
+     * @throws DukeException No current save file.
+     */
+    public ArrayList<String> load() throws DukeException {
```

Comment:
Perhaps you could name this method loadTasks, since it returns a collection of tasks?